### DISCUSSION TITLE
RTL logical properties does not compile

### DISCUSSION THREAD

**Discussion by nasdaq-dubai-dfm** [tailwindlabs/tailwindcss#11092] [Help] [answered]
Hey there, 
no clue what i am doing wrong, but after updating tailwind to v3.3 and remove the third party "tailwind-rtl" plugin, the logical properties doesnt render the way it should. For eg:h

When i use ms-auto, instead of rendering margin-inline-start, tailwind render margin-left.
The html has dir-rtl attribute, i double checked.

.ms-auto {
    margin-left: auto;
}

what am I doing wrong? None of the logical properties work in my setup.
This is a Nuxt 2 project.

Any help is highly appreciated.

**Comment by nasdaq-dubai-dfm** [comment, marked as answer]
Resolved. Downgraded Nuxt to 2.15.7 and the issue is resolved

**Comment by dohahelmy** [comment]
No need to downgrade
you need to add `stage: 3` to postcss preset in `build` in your `nuxt.config.js`  and it should work
![image](https://github.com/tailwindlabs/tailwindcss/assets/17557397/cda238f3-5fed-49ed-a9a3-38fb63dbebb2)
